common4j/src/main/com/microsoft/identity/common/java/nativeauth/controllers/results/INativeAuthCommandResult.kt

```diff
@@ -62,7 +70,19 @@ interface INativeAuthCommandResult {
         SignUpSubmitCodeCommandResult, SignUpResendCodeCommandResult,
         SignUpSubmitPasswordCommandResult,
         ResetPasswordStartCommandResult, ResetPasswordSubmitCodeCommandResult,
-        ResetPasswordResendCodeCommandResult, ResetPasswordSubmitNewPasswordCommandResult
+        ResetPasswordResendCodeCommandResult, ResetPasswordSubmitNewPasswordCommandResult {
+        override fun toSafeString(mayContainPii: Boolean): String {
+            return if (mayContainPii) {
+                "UnknownError(correlationId=$correlationId, error=$error, errorDescription=$errorDescription), details=$details, errorCodes=$errorCodes"
+            } else {
+                "UnknownError(correlationId=$correlationId)"
```

Comment:
errorCodes are numbers and do not contain PII so they should be part of both outputs.

Reply:
They're not logged by us currently (in the API layer). We based this on MSAL's approach, who don't log errors. I'll check with MSAL folks to get their input.